test/system/src/test/java/io/pravega/test/system/ControllerFailoverTest.java

```diff
@@ -179,10 +190,109 @@ public void failoverTest() throws InterruptedException, ExecutionException {
         Assert.assertEquals(2, streamSegments.getSegments().size());
     }
 
+    @Test
+    public void sweepOverTest() throws InterruptedException, ExecutionException {
+        String scope = "testSweepOverScope" + RandomStringUtils.randomAlphabetic(5);
+        String stream = "testSweepOverStream" + RandomStringUtils.randomAlphabetic(5);
+        String readerGroupName = "testSweepOverRG" + RandomStringUtils.randomAlphabetic(5);
+        int initialSegments = 1;
+        List<Long> segmentsToSeal = Collections.singletonList(0L);
+        Map<Double, Double> newRangesToCreate = new HashMap<>();
+        newRangesToCreate.put(0.0, 1.0);
+
+        ClientConfig clientConfig = Utils.buildClientConfig(controllerURIDirect);
+        // Connect with controller instance
+        final Controller controller = new ControllerImpl(
+                ControllerImplConfig.builder()
+                        .clientConfig(clientConfig)
+                        .build(), executorService);
+
+        // Create scope, stream, and a transaction with high timeout value.
+        controller.createScope(scope).join();
+        log.info("Scope {} created successfully", scope);
+
+        createStream(controller, scope, stream, ScalingPolicy.fixed(initialSegments));
+        log.info("Stream {}/{} created successfully", scope, stream);
+
+        StreamImpl stream1 = new StreamImpl(scope, stream);
+
+        // Initiate scale operation. It will block until ongoing transaction is complete.
```

Comment:
I do not think this logline is correct due to rolling transactions now enabled on the controller.

Reply:
it's removed now, thank you :)